src/components/Calculator.js

```diff
@@ -1,74 +1,111 @@
-        <button id="btn-percentage" className="btn" type="button">
-          %
-        </button>
-        <button id="btn-one" className="btn" type="button">
-          1
-        </button>
-        <button id="btn-two" className="btn" type="button">
-          2
-        </button>
-        <button id="btn-three" className="btn" type="button">
-          3
-        </button>
-        <button id="btn-four" className="btn" type="button">
-          4
-        </button>
-        <button id="btn-five" className="btn" type="button">
-          5
-        </button>
-        <button id="btn-six" className="btn" type="button">
-          6
-        </button>
-        <button id="btn-seven" className="btn" type="button">
-          7
-        </button>
-        <button id="btn-eight" className="btn" type="button">
-          8
-        </button>
-        <button id="btn-nine" className="btn" type="button">
-          9
-        </button>
-        <button id="btn-zero" className="btn" type="button">
-          0
-        </button>
-        <button id="btn-period" className="btn" type="button">
-          .
-        </button>
-      </section>
+const Calculator = () => {
+  const [state, setState] = useState({ total: null, next: null, operation: null });
 
-      <section className="calc-operators">
-        <button id="btn-four" className="btn" type="button">
-          /
-        </button>
-        <button id="btn-four" className="btn" type="button">
-          *
-        </button>
-        <button id="btn-four" className="btn" type="button">
-          -
-        </button>
-        <button id="btn-four" className="btn" type="button">
-          +
-        </button>
-        <button id="btn-four" className="btn" type="button">
-          =
-        </button>
-      </section>
-    </div>
-  </main>
-);
+  const handleClick = (buttonName) => {
+    const newState = calculate(state, buttonName);
+    setState(newState);
+  };
+
+  const handleEquals = () => {
+    const { total, next, operation } = state;
+    if (total && next && operation) {
+      const result = operate(Big(total), Big(next), operation);
+      setState({ total: result.toString(), next: null, operation: null });
+    }
+  };
+
+  const { total, next, operation } = state;
+  return (
+    <main className="calculator">
+      <header className="calc-output">
+        {total}
+        {' '}
+        {operation}
+        {next}
+      </header>
+
+      <div className="calc-grid">
+        <section className="calc-numbers">
+          <button id="btn-ac" className="button_calculator" type="button" onClick={() => handleClick('AC')}>
+            AC
+          </button>
+          <button id="btn-minus-plus" className="button_calculator" type="button" onClick={() => handleClick('+/-')}>
+            +/-
+          </button>
+          <button id="btn-percentage" className="button_calculator" type="button" onClick={() => handleClick('%')}>
+            %
+          </button>
```

Comment:
- [OPTIONAL] As requested by the previous reviewer 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻